lesson_2/Lesson_2.2.ipynb

```diff
@@ -0,0 +1,204 @@
+{
```

Comment:
<p><em>We don't know </em><strong><em>the true distribution</em></strong><em>, but by taking samples in a clever way we can estimate it</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>